spec/models/dossier_spec.rb

```diff
@@ -1126,6 +1126,33 @@
     end
   end
 
+  describe '#notify_draft_not_submitted' do
+    let!(:user1) { create(:user) }
+    let!(:user2) { create(:user) }
+    let!(:procedure_closed_in_time) { create(:procedure, auto_archive_on: Time.zone.today + Dossier::REMAINING_TIME_BEFORE_CLOSING) }
+    let!(:procedure_closed_later) { create(:procedure, auto_archive_on: Time.zone.today + Dossier::REMAINING_TIME_BEFORE_CLOSING + 1.day) }
+    let!(:procedure_closed_before) { create(:procedure, auto_archive_on: Time.zone.today + Dossier::REMAINING_TIME_BEFORE_CLOSING - 1.day) }
+
+    # user 1 has three draft dossiers where one is for procedure that closes in two days ==> should trigger one mail
+    let!(:draft_in_time) { create(:dossier, user: user1, procedure: procedure_closed_in_time) }
```

Comment:
Pinaillage : pour clarifier, je propose :

```suggestion
    let!(:draft_near_closing) { create(:dossier, user: user1, procedure: procedure_closed_in_time) }
```